stylesheets/components/_button.scss

```diff
@@ -1,3 +1,60 @@
 // -----------------------------------------------------------------------------
 // This file contains all styles related to the button component.
 // -----------------------------------------------------------------------------
+  .button {
+    display: flex;
+    justify-content: center;
+    align-items: center;  
+    flex: 0 0 auto;
+    border-radius: 4px;
+    
+    &:hover {
+        cursor: pointer;
+        box-shadow:(0 0 2px 1px $color-text-footer);
+        color: $color-button-text;
+        background-color: $color-text-footer;        
+      }
+      
+      &--yellow {
+        font-family: $font-family-basic;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 16px;
+        line-height: 24px;
+        color: $color-button-text;  
+        background: $color-button;
+        border-color: $color-button;
+        border-style: solid;
+
+        &:hover{
+            background-color: $color-button-text;
+            color: $color-button;
+            box-shadow:(0 0 2px 1px $color-button); 
+        }
+      }
+  }
+  
+
+  .nav__button-sign-in {
```

Comment:
Do not create specific buttons, etc 

Create common re-usable components.

In the current design buttons are the same with 3 differences:
- primary color button
- secondary color button
- large button


User these differences as modifications

Reply:
completed